src/action.js

```diff
@@ -94,23 +94,23 @@ function isAMajorReleaseBump(change) {
   const from = change.delete
   const to = change.insert
 
-  if (!from || !to) {
-    return false
-  }
-
   const diff = semverDiff(semverCoerce(from), semverCoerce(to))
   return diff === targetOptions.major
 }
 
 function parsePrTitle(pullRequest) {
-  const expression = /bump (\S+) from (\S+) to (\S+)/i
+  const expression = /bump \S+ from (\S+) to (\S+)/i
   const match = expression.exec(pullRequest.title)
 
   if (!match) {
-    return {}
+    throw new Error('Error while parsing PR title, expected: `bump <package> from <old-version> to <new-version>`')
   }
+  const [, oldVersion, newVersion] = match
 
-  const [, packageName, oldVersion, newVersion] = match
+  // Get package name from branch
+  // dependabot branch names are in format "dependabot/npm_and_yarn/pkg-0.0.1"
+  // or "dependabot/github_actions/fastify/github-action-merge-dependabot-2.6.0"
+  const packageName = pullRequest.head.ref.split('/').pop().split('-').slice(0, -1).join('-')
```

Comment:
I'd put this logic in a function like `getPackageName(pullRequest.head.ref)` or something, so it can be unit tested massively and easily

Reply:
Example PR for invalid title
https://github.com/austins/smoothnanners-web/pull/17
` dependabot/npm_and_yarn/react-and-react-dom-18.0.0`
https://github.com/wallabag/wallabag/pull/5537
`dependabot/npm_and_yarn/stylelint-and-stylelint-config-standard-and-stylelint-scss-14.2.0]`

Reply:
> Example PR for invalid title [austins/smoothnanners-web#17](https://github.com/austins/smoothnanners-web/pull/17) ` dependabot/npm_and_yarn/react-and-react-dom-18.0.0` [wallabag/wallabag#5537](https://github.com/wallabag/wallabag/pull/5537) `dependabot/npm_and_yarn/stylelint-and-stylelint-config-standard-and-stylelint-scss-14.2.0`

Those are actually not affected by this I don't think. Since they're package.json changes they will be parsed from the diff [here](https://github.com/fastify/github-action-merge-dependabot/blob/main/src/moduleVersionChanges.js#L44). This function will only apply to changes that are not on package.json (e.g. github workflow files)